### DISCUSSION TITLE
Custom plugin missing style

### DISCUSSION THREAD

**Discussion by HoaPhamThi** [JiHong88/SunEditor#1345] [Q&A]
When I add a custom plugin with the command, my styles are shown in the editor but when I call getContents() method to get contents, they are all deleted, only the tag names remain. I tried with the demo code but this error still occurs. Please help me asap. Thanks.
http://suneditor.com/sample/html/customPlugins.html

**Comment by JiHong88** [comment]
What tags, classes and styles do you use?
Please also let me know the format of the tag.
If it does not fit the format set in the editor, it will be edited or deleted.
Options include "addTagsWhitelist", "tagsBlacklist", "attributesBlacklist", and "attributesWhitelist" etc.
https://github.com/JiHong88/suneditor?tab=readme-ov-file#options
